style.css

```diff
@@ -85,11 +87,264 @@ ul {
 }
 
 .social-link {
-  margin-top: 5vh;
+  margin-top: 10vh;
   width: 50%;
 }
 
 li > a {
   font-size: 1.5rem;
   color: #505f79;
 }
+
+.bottom-arrow {
+  display: flex;
+  justify-content: center;
+}
+
+.bottom {
+  display: inline-block;
+  width: 4em;
+  height: 4em;
+  border: 0.08em solid #dfe1e6;
+  border-radius: 50%;
+  margin: auto;
+}
+
+.bottom::after {
+  position: relative;
+  left: 22px;
+  top: 8px;
+  content: '';
+  display: inline-block;
+  margin-top: 0.6em;
+  width: 1em;
+  height: 0.9em;
+  border-top: 0.2em solid #333;
+  border-right: 0.2em solid #333;
+  border-radius: 3px;
+  -moz-transform: rotate(135deg);
+  -webkit-transform: rotate(135deg);
+  transform: rotate(135deg);
+}
+
+.works-title {
+  font-size: 2.5rem;
+  text-align: center;
+  letter-spacing: 1px;
+  width: 80%;
+  font-weight: 900;
+  color: #172b4d;
+  margin: 3vh auto 5vh auto;
+}
+
+.works-title::after {
+  border-radius: 10px;
+  content: "";
+  width: 48px;
+  display: block;
+  height: 4px;
+  background-color: #32b37e;
+  margin: auto;
+  margin-top: 20px;
+}
+
+.projects-container {
+  display: grid;
+  grid-template-rows: auto;
+  column-gap: 10px;
+}
+
+.project {
+  background-color: #ebf0ee;
+  width: 90%;
+  margin: 2vh auto;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #d0d9d4;
+}
+
+.project-image {
+  overflow: hidden;
+  width: 100%;
+  height: 300px;
+  border-radius: 5px;
+  background-size: auto;
+  background-position: center;
+}
+
+.project-description {
+  border-radius: 10px;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.project-description h3 {
+  font-size: 1.5rem;
+  text-align: center;
+  letter-spacing: 1px;
+  font-weight: 900;
+  color: #3a4a42;
+  margin-top: 20px;
+}
+
+.project-description ul {
+  padding: 10px;
+  margin: auto;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.project-description ul > li {
+  font-size: 12px;
+  font-weight: 500;
+  margin: 5px 0;
+  padding: 10px;
+  background-color: #ebf0ee;
+  border-radius: 5px;
+}
+
+.project-description > .btn {
+  border-radius: 5px;
+  font-size: 1.3rem;
+  text-align: center;
+  text-decoration: none;
+  padding: 15px;
+  background-color: #32b37e;
+  color: white;
+  margin: 15px 0 25px 0;
+}
+
+.about-section {
+  display: block;
+}
+
+.about-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  margin: auto;
+}
```

Comment:
- [ ] Kindly add the background image, review the requirement above. 👆🏽 
- Figma template
![image](https://user-images.githubusercontent.com/58642949/185605749-f032d437-5b1e-4b12-916e-cfe60f59b166.png)

- Yours:
![image](https://user-images.githubusercontent.com/58642949/185605970-bb3b99ba-5b7f-4489-a6c2-80b835af374f.png)